File: back/CdfsPalomasAPI/Models/Partido.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace CdfsPalomasAPI.Models;
[Table("partidos")]
public class Partido
{
    public int Id { get; set; }
    public DateTime Fecha { get; set; }
    public string Rival { get; set; } = null!;

    [Column("goles_favor")]
    public int GolesFavor { get; set; } = 0;

    [Column("goles_contra")]
    public int GolesContra { get; set; } = 0;

    [Column("es_local")]
    public bool EsLocal { get; set; } = true;

    [Column("id_competicion")]
    public int IdCompeticion { get; set; }
    public bool Jugado { get; set; } = false;
    public Competicion Competicion { get; set; } = null!;
    public ICollection<Convocatoria> Convocatorias { get; set; } = new List<Convocatoria>();
    public ICollection<Galeria> Galerias { get; set; } = new List<Galeria>();
}

File: front/src/app/core/models/partido.ts
export interface Partido {
  id: number;
  fecha: string;
  rival: string;
  golesFavor: number;
  golesContra: number;
  esLocal: boolean;
  idCompeticion: number;
  nombreCompeticion: string;
  categoria: string;
  nombreTemporada: string;
  jugado: boolean;
}

export interface CrearPartido {
  fecha: string;
  rival: string;
  golesFavor: number;
  golesContra: number;
  esLocal: boolean;
  idCompeticion: number;
  jugado: boolean;
}